app/pages/project/home/metadata.spec.js

```diff
@@ -1,16 +1,14 @@
 import React from 'react';
 import assert from 'assert';
-import sinon from 'sinon';
-import { render, mount } from 'enzyme';
+import { render } from 'enzyme';
 
 import ProjectMetadata from './metadata';
 
-describe('ProjectMetadata', function(){
+describe('ProjectMetadata', () => {
```

Comment:
Not a blocking comment but Mocha discourages arrow functions so we prefer `function () {…}` for anonymous functions in tests. https://mochajs.org/#arrow-functions

Reply:
great catch, didn't have my eslint autoFixOnSave set as I would have liked, thanks